2019-12-11 12:59:00,829 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-12 13:54:52,088 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-13 14:24:04,950 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-13 14:26:39,735 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-13 14:38:46,006 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-13 14:39:17,905 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-13 14:41:16,028 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-13 14:42:16,210 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:07:42,833 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:09:57,504 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:11:14,057 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:20:55,733 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:21:57,267 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:25:38,825 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:29:07,437 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:30:47,952 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:32:17,357 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:39:20,180 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 14:41:51,177 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 21:38:35,384 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 21:45:44,389 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 22:02:00,569 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 22:03:02,650 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 23:43:53,561 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 23:45:46,391 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 23:46:29,507 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 23:48:29,444 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 23:48:55,154 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-14 23:49:22,123 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-15 00:23:19,025 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-15 00:24:03,925 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-15 00:25:04,064 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-16 13:45:04,820 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-16 13:53:29,800 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-16 14:10:36,497 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-16 14:11:28,912 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-16 14:22:24,784 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:47]
2019-12-17 14:06:39,428 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:49]
2019-12-17 14:09:39,233 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:49]
2019-12-17 14:18:52,833 INFO: Microblog startup [in C:\Users\duvolle\Desktop\microblog\app\__init__.py:49]
2019-12-17 14:18:57,084 ERROR: Exception on /user/tumysov [GET] [in c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\duvolle\Desktop\microblog\app\routes.py", line 129, in user
    next_url=next_url, prev_url=prev_url)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\duvolle\Desktop\microblog\app\templates\user.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\duvolle\Desktop\microblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block "html"
    {% block body -%}
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block "body"
    {% block content -%}
  File "C:\Users\duvolle\Desktop\microblog\app\templates\base.html", line 48, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\duvolle\Desktop\microblog\app\templates\user.html", line 24, in block "app_content"
    {% include '_post.html' %}
  File "C:\Users\duvolle\Desktop\microblog\app\templates\_post.html", line 12, in top-level template code
    said {{ moment(post.timestamp).fromNow('LLL') }}:
  File "c:\users\duvolle\downloads\opendatascience\ods\lib\site-packages\flask_moment.py", line 133, in fromNow
    return self._render("fromNow(%s)" % int(no_suffix), refresh)
ValueError: invalid literal for int() with base 10: 'LLL'
